src/style.css

```diff
@@ -1,3 +1,89 @@
 body {
   margin: 0;
+  background-color: beige;
+
+  --color: #00acdc;
+}
+
+.header {
+  display: flex;
+  flex-shrink: 0;
+  align-items: center;
+  justify-content: space-between;
+  color: #000;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  line-height: normal;
+  text-transform: uppercase;
+  background-color: #fff;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.img {
+  margin-left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.nav {
+  position: relative;
+  display: block;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__link {
+  display: inline-block;
+  margin: 0 10px;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+  color: #000;
+}
+
+.nav li a:hover {
+  color: var(--color);
+}
+
+.is-active {
+  color: var(--color);
+  margin-left: 0;
+}
+
+.link-last {
+  margin-right: 0;
+}
+
+.button {
+  position: absolute;
+  width: 37px;
+  height: 4px;
+  background-color: var(--color);
+  margin-top: 56px;
+  border-radius: 8px;
+  border-color: var(--color) ;
+}
+
+.hover::after {
+  text-decoration: none;
+}
+
+.nav__link__four {
+  margin-right: 0;
+  margin-left: 0;
+}
+
+.nav__link__tree {
+  margin-right: 20px;
+}
+
+.nav__link__five {
+  margin-left: 20px;
 }
```

Comment:
not fixed:
![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/f7daca98-2f47-48d8-86c0-8398d2674ec7)